conf/th/thailand01.yml

```diff
@@ -390,58 +390,58 @@ years:
     date: '2021-01-01'
     names:
       en: New Years Day
-      th: วันขึ้นปีใหม่
```

Comment:
🤔

Reply:
Ah that was an error, I was supposed to add in the EN name here rather than filly delete that line. Sorry, I'm unfamiliar with how this review & comment functionality works - do I go back in and edit this again, or as the approver are you able to?

Reply:
Oh, no you have done that. Sorry, by this unhelpful comment I just meant are we sure we want to remove the Thai, which you've answered for me!